src/lib.rs

```diff
@@ -324,6 +324,43 @@ pub struct WalletOpts {
     #[cfg(feature = "esplora")]
     #[structopt(flatten)]
     pub esplora_opts: EsploraOpts,
+    #[cfg(feature = "compact_filters")]
+    #[structopt(flatten)]
+    pub compactfilter_opts: CompactFilterOpts,
+}
+
+/// Compact Filter options
+///
+/// Compact filter peer information used by [`OnlineWalletSubCommand`]s.
+#[cfg(feature = "compact_filters")]
+#[derive(Debug, StructOpt, Clone, PartialEq)]
+pub struct CompactFilterOpts {
+    /// Sets the SOCKS5 proxy for the Electrum client
```

Comment:
The doc comment here should be something like:
Sets the SOCKS5 proxy for the compact-filter blockchain client

Reply:
Fixed.